web/index.html

```diff
@@ -41,44 +42,137 @@
 
   <title>Tvarkau Lietuvą</title>
   <script type="text/javascript" src="https://alcdn.msauth.net/browser/2.13.1/js/msal-browser.min.js"
-          integrity="sha384-2Vr9MyareT7qv+wLp1zBt78ZWB4aljfCTMUrml3/cxm0W81ahmDOC6uyNmmn0Vrc"
-          crossorigin="anonymous"></script>
+    integrity="sha384-2Vr9MyareT7qv+wLp1zBt78ZWB4aljfCTMUrml3/cxm0W81ahmDOC6uyNmmn0Vrc"
+    crossorigin="anonymous"></script>
   <script src="assets/packages/aad_oauth/assets/msalv2.js"></script>
   <link rel="manifest" href="manifest.json">
   <script>
     // The value below is injected by flutter build, do not touch.
     var serviceWorkerVersion = null;
   </script>
+
   <!-- This script adds the flutter initialization JS code -->
-  <script src="flutter.js" defer></script>
+  <script src="flutter.js" defer=""></script>
   <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyB5YFBt24Abvt16M3g-aBtIlTCAM2G6fBU"></script>
-  <script async src="https://www.googletagmanager.com/gtag/js?id=G-ZNCF3S3TMN"></script>
+  <script async="" src="https://www.googletagmanager.com/gtag/js?id=G-ZNCF3S3TMN"></script>
   <script>
-  window.dataLayer = window.dataLayer || [];
-  function gtag(){dataLayer.push(arguments);}
-  gtag('js', new Date());
-  gtag('config', 'G-ZNCF3S3TMN');
-</script>
+    window.dataLayer = window.dataLayer || [];
+    function gtag() { dataLayer.push(arguments); }
+    gtag('js', new Date());
+    gtag('config', 'G-ZNCF3S3TMN');
+  </script>
   <script src="//cdnjs.cloudflare.com/ajax/libs/pdf.js/2.4.456/pdf.min.js"></script>
   <script type="text/javascript">
-   pdfjsLib.GlobalWorkerOptions.workerSrc = "//cdnjs.cloudflare.com/ajax/libs/pdf.js/2.4.456/pdf.worker.min.js";
-</script>
+    pdfjsLib.GlobalWorkerOptions.workerSrc = "//cdnjs.cloudflare.com/ajax/libs/pdf.js/2.4.456/pdf.worker.min.js";
+  </script>
+
+  <meta content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no" name="viewport">
+
+
```

Comment:
@tupats, do you have any suggestions on maintaining consistent formatting for HTML files? Ignas and I both use Android Studio. Any insights or recommendations would be appreciated.

Reply:
I don't have experience with this, my team and I are using vscode and usually we don't touch HTML files, sorry. I will disable .html formatting for this project.